pants-plugins/src/python/internal_backend/sitegen/tasks/sitegen.py

```diff
@@ -251,22 +251,36 @@ def generate_site_toc(config, precomputed, here):
 
   def recurse(tree, depth_so_far):
     for node in tree:
+      if 'collapsible_heading' in node and 'pages' in node:
+        heading = node['collapsible_heading']
+        pages = node['pages']
+        links = []
+        collapse_open = False
+        for cur_page in pages:
+          html_filename = '{}.html'.format(cur_page)
+          page_is_here = cur_page == here
+          if page_is_here:
+            link = html_filename
+            collapse_open = True
+          else:
+            link = os.path.relpath(html_filename, os.path.dirname(here))
+          links.append(dict(link=link, text=precomputed.page[cur_page].title, here = page_is_here))
+        site_toc.append(dict(depth = depth_so_far, links = links, dropdown = True, heading = heading, id = heading.replace(' ','-'), open = collapse_open))
```

Comment:
nit: there shouldn't be whitespace around the `=`s in the kwargs in the `dict` calls here.
ie
`call(x=y, z=a)`
instead of 
`call(x = y, z = a)`